cocos2dx/menu_nodes/CCMenu.h

```diff
@@ -141,27 +141,27 @@ class CC_DLL CCMenu : public CCLayer, public CCRGBAProtocol
     void setHandlerPriority(int newPriority);
 
     //super methods
-    virtual void addChild(CCNode * child);
-    virtual void addChild(CCNode * child, int zOrder);
-    virtual void addChild(CCNode * child, int zOrder, int tag);
-    virtual void registerWithTouchDispatcher();
+    virtual void addChild(CCNode * child) override;
+    virtual void addChild(CCNode * child, int zOrder) override;
+    virtual void addChild(CCNode * child, int zOrder, int tag) override;
+    virtual void registerWithTouchDispatcher() override;
 
     /**
     @brief For phone event handle functions
     */
-    virtual bool ccTouchBegan(CCTouch* touch, CCEvent* event);
-    virtual void ccTouchEnded(CCTouch* touch, CCEvent* event);
-    virtual void ccTouchCancelled(CCTouch *touch, CCEvent* event);
-    virtual void ccTouchMoved(CCTouch* touch, CCEvent* event);
+    virtual bool ccTouchBegan(CCTouch* touch, CCEvent* event) override;
+    virtual void ccTouchEnded(CCTouch* touch, CCEvent* event) override;
+    virtual void ccTouchCancelled(CCTouch *touch, CCEvent* event) override;
+    virtual void ccTouchMoved(CCTouch* touch, CCEvent* event) override;
 
     /**
     @since v0.99.5
     override onExit
     */
-    virtual void onExit();
+    virtual void onExit() override;
 
-    virtual void setOpacityModifyRGB(bool bValue) {CC_UNUSED_PARAM(bValue);}
-    virtual bool isOpacityModifyRGB(void) { return false;}
+    virtual void setOpacityModifyRGB(bool bValue) override {CC_UNUSED_PARAM(bValue);}
+    virtual bool isOpacityModifyRGB(void) const override { return false; }
     
     virtual bool isEnabled() { return m_bEnabled; }
```

Comment:
Yeah. I didn't do all of them, I went through the ones that permeate through Griffin.